manager/sql/databases.go

```diff
@@ -0,0 +1,276 @@
+package sql
+
+import migrate "github.com/rubenv/sql-migrate"
+
+type Statements struct {
+	Migrations                    *migrate.MemoryMigrationSource
+	QueryInsertPolicy             string
+	QueryInsertPolicyActions      string
+	QueryInsertPolicyActionsRel   string
+	QueryInsertPolicyResources    string
+	QueryInsertPolicyResourcesRel string
+	QueryInsertPolicySubjects     string
+	QueryInsertPolicySubjectsRel  string
+	FindRequestCandidates         string
+}
+
+var Databases = map[string]Statements{
+	"postgres": {
+		Migrations: &migrate.MemoryMigrationSource{
+			Migrations: []*migrate.Migration{
+				{
+					Id: "1",
+					Up: []string{
```

Comment:
I think it makes sense to use the same up/down statements wherever possible which in this case would be that postgres and mysql have the same up/down statements for id 1. What do you think?

Reply:
When I was reorganizing with the goal of adding official support for more databases, it made more sense to me to have each database's statements remain separate for readability. I can revert this since the library will only be supporting MySQL & PostgreSQL.